pkg/collector/python/init.go

```diff
@@ -303,31 +303,34 @@ func pathToBinary(name string, ignoreErrors bool) (string, error) {
 }
 
 func resolvePythonExecPath(ignoreErrors bool) (string, error) {
-	// Since the install location can be set by the user on Windows we use relative import
-	if runtime.GOOS == "windows" {
-		_here, err := executable.Folder()
+	// Allow to relatively import python
```

Comment:
Why is this change showing up here? It is from https://github.com/DataDog/datadog-agent/pull/31426 and it was merged already as far as I can see

Reply:
Github didn't update it, I think an empty commit should remove it